### ISSUE TITLE
Just wanted to say: THANKS SO MUCH FOR THIS PACKAGE

### ISSUE THREAD

**Issue by anonimitoraf** [misohena/el-easydraw#10]
I've been looking for a simple drawing tool within Emacs! I was contemplating on making one myself, until I came across this package.

Question: Are there plans to add reusable widgets like arrowed lines, etc? I'm happy to help with implementing if there are plans to do so

**Comment by misohena** [comment]
anonimitorafさんこんにちは。
reusable widgetsというのがどのようなものか分からないのですが教えて頂けないでしょうか。
何か定義済みの図形を挿入する機能か何かでしょうか。

(Google Translate):
Hello anonimitoraf.
I don't know what reusable widgets are, can you tell me what they are?
Is it the ability to insert some predefined shapes or something?

**Comment by anonimitoraf** [comment]
Yep, that's right. For example, I personally tend to draw arrows 
![image](https://user-images.githubusercontent.com/15933322/194022128-206c540a-2077-4097-a39c-4d79d3172ec6.png)
for system architecture diagrams

**Comment by misohena** [comment]
こんにちは。返事が遅くなってすみません。
それは良いハックだと思いましたが、以前から計画していたカスタムシェイプ機能を実装している所です。
まだ修正するところは残っていますが、その方法よりは使いやすいと思います。
エディタ上でコピーした図形をシェイプピッカーへペーストできます。

(Google Translate)
hello. Sorry for replying late.
I thought it was a good hack, but I'm implementing the custom shapes feature that I've been planning for a long time.
I still have some things to fix, but I think it's easier to use than that method.
You can paste the shape copied on the editor to the shape picker.

![screenshot-20221208-custom-shape-test](https://user-images.githubusercontent.com/250549/206906917-366cf715-77d0-4517-a27f-7ffbc5b0921b.png)
